client/src/pages/SplashPage/SplashPage.js

```diff
@@ -15,10 +16,21 @@ class SplashPage extends React.Component {
   toRegister() {
     this.props.history.push("/register");
   }
+
+  renderRedirect = () => {
+    if (localStorage.getItem("token") != undefined) {
+      return <Redirect to="/home" />;
+    } else{
+      console.log("Not logged in")
+    }
+  };
```

Comment:
This logic should probably be part of the parent component. If the user is logged in (and there's a token) then we want to just render the home page and not render the login page. here, you're rendering the login page, then checking to see if the user is logged in, and if they logged in only then redirecting them to the home page.